lib/DocumentManagerInterface.php

```diff
@@ -122,4 +124,56 @@ public function createQuery($query, $locale = null, array $options = []);
      * http://doctrine-phpcr-odm.readthedocs.org/en/latest/reference/query-builder.html
      */
     public function createQueryBuilder();
+
+    /**
+     * Retrieve the document inspector.
+     *
+     * NOTE: In the future this will be a factory for retrieving inspectors
+     * for a given object/class.
```

Comment:
We might be able to do that generally - providing that nothing outside of hte document manager needs it (not currently the case as it is required by the `ContentMapper`.